astropy/constants/tests/test_constant.py

```diff
@@ -65,7 +64,11 @@ def test_e():
     assert isinstance(e.gauss, Q)
     assert isinstance(e.esu, Q)
 
-    assert e.si * E == Q(100, 'eV/m')
+    # We cannot use == comparison because:
+    # e.si * E = <Quantity 1.60217663e-17 C V / m>
+    # (e.si * E).to(u.eV / u.m) = <Quantity 100.00000082 eV / m>
+    assert_quantity_allclose(e.si * E, Q(100, 'eV/m'))
```

Comment:
I think this is because in CODATA2018, we now mathematically derive some of the constants in real time instead of pre-calculating it "on paper" and hardcoding the answer.